packages/angular/cli/models/command-runner.ts

```diff
@@ -70,18 +101,27 @@ export async function runCommand(commandMap: CommandMap,
     ? CommandScope.inProject
     : CommandScope.outsideProject;
 
-  let Cmd: CommandConstructor | null;
-  Cmd = commandName ? findCommand(commandMap, commandName) : null;
+  const commandMapPath = findUp('commands.json', __dirname);
```

Comment:
This means if we internally want to have our own set of commands, we have to maintain a commands.json file under external CLI repo. Any way to avoid this?